src/context/auth/authContextController/AuthContextController.tsx

```diff
@@ -65,13 +66,13 @@ export const AuthContextController = ({ children }: AuthContextControllerProps)
   const value: AuthContextValue = useMemo(
     () => ({
       ...state,
-      isAuthenticating: isAuthenticating || isLoadingUser,
+      isAuthenticating: isAuthenticating || (isLoadingUser && fetchStatus !== 'idle'),
```

Comment:
Maybe we can consider add to our RQ abstraction some boolean which will be aqual to 
`isLoadingUser && fetchStatus !== 'idle'`

Reply:
Hm I can imagine it's being abstracted like this:

```tsx
export const useQuery = <TData = unknown, TError = unknown>(
  queryKey: QueryKey,
  options?: UseQueryOptions<TData, TError>,
): UseQueryResult<TData, TError> & { isLoadingAndEnabled: boolean } => {
  const { queryFn } = useApiClient();
  const _queryFn = useMemo(() => queryFn<TData>(), [queryFn]);
  const result = useRqQuery<TData, TError, TData, QueryKey>(queryKey, _queryFn, options);

  return { ...result, isLoadingAndEnabled: result.isLoading && result.fetchStatus !== 'idle' };
};
```

Reply:
Yes, I thought about that kind of abstraction

Reply:
done